disorder-fsm/src/Disorder/FSM.hs

```diff
@@ -1,9 +1,14 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
Cheers, @tranma need to keep up with my ghc releases and reading.
@michaelneale https://typesandkinds.wordpress.com/2013/08/15/roles-a-new-feature-of-ghc/